<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listGen">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>256</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listQ">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>256</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listCuns">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>10</y>
      <width>256</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGen">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>410</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Start/Stop 1 thread</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCons">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>400</y>
      <width>151</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Start/Stop 3 thread</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnExit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>440</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>EXIT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>390</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
